### PR TITLE
add new crds created by ceph csi operation

### PR DESCRIPTION
**Pull request by yati1998**
this commit collects new crds details which
are newly created by ceph-csi-operator

### CODE REVIEW

File: collection-scripts/gather_namespaced_resources [hunk 1]
```diff
@@ -91,6 +91,11 @@ oc_get_all+=("networkfence")
 oc_get_all+=("network-attachment-definitions")
 oc_get_all+=("encryptionkeyrotationjob")
 oc_get_all+=("encryptionkeyrotationcronjob")
+oc_get_all+=("operatorconfig")
+oc_get_all+=("driver")
+oc_get_all+=("cephconnection")
+oc_get_all+=("clientprofile")
+oc_get_all+=("clientprofilemapping")
```
Comment (Madhu-1) [inline collection-scripts/gather_namespaced_resources:98]: please use the full names as these are very generic names we might get into problem, if we have duplicate CRD with name short names

**Review by Madhu-1 — APPROVED** [review]

**Comment by yati1998** [comment]
@black-dragon74 please review the PR.

**Pull request merged**